modules/schema/validatecreate_test.go

```diff
@@ -0,0 +1,383 @@
+package schema
+
+import (
+	"errors"
+	"reflect"
+	"testing"
+
+	"github.com/spaceuptech/space-cloud/config"
+	"github.com/spaceuptech/space-cloud/model"
+	"github.com/spaceuptech/space-cloud/modules/crud"
+)
+
+var queries = `
+ type tweet {
+ 	id: ID @id
+ 	createdAt: DateTime@createdAt
+ 	text: String
+ 	owner: [String]
+ 	location: location@foreign
+	age : Float!
+	isMale:Boolean
+	exp:Integer
+	event:event_logs
+	person : sharad @link(table:sharad, from:Name, to:isMale)
+   }
+
+   type test {
+	id : ID @id
+	person : sharad @link(table:sharad, from:Name, to:isMale)
+   }
+
+   type location {
+ 	id: ID! @id
+ 	latitude: Float
+ 	longitude: Float
+
+   }
+   type sharad {
+ 	  Name : String!
+ 	  Surname : String!
+ 	  age : Integer!
+ 	  isMale : Boolean!
+ 	  dob : DateTime@createdAt
+   }
+   type event_logs {
+ 	id: Integer
+ 	name: String
+   }
+ `
+var Parsedata = config.Crud{
+	"mongo": &config.CrudStub{
+		Collections: map[string]*config.TableRule{
+			"tweet": &config.TableRule{
+				Schema: queries,
+			},
+			"test": &config.TableRule{
+				Schema: queries,
+			},
+			"location": &config.TableRule{
+				Schema: queries,
+			},
+		},
+	},
+}
+
+func TestSchema_ValidateCreateOperation(t *testing.T) {
+
+	tdd := []struct {
+		dbName, coll, description string
+		value                     model.CreateRequest
+		want                      error
+	}{
+		{
+			dbName: "sqlserver",
+			coll:   "tweet",
+			want:   errors.New("No db was found named sqlserver"),
```

Comment:
change in every case want accordingly to bool,
if the data provided in test case is such that error will occur then make it false otherwise true

Reply:
remove the hard coding of error in all the test cases